js_modules/dagster-ui/packages/ui-core/src/ticks/__fixtures__/SensorDryRunDialog.fixtures.tsx

```diff
@@ -177,3 +180,161 @@ export const PersistCursorValueMock: MockedResponse<SetSensorCursorMutation> = {
     },
   },
 };
+
+export const SensorLaunchAllMutation: MockedResponse<LaunchMultipleRunsMutation> = {
+  request: {
+    query: LAUNCH_MULTIPLE_RUNS_MUTATION,
+    variables: {
+      executionParamsList: [
+        {
+          runConfigData:
+            'solids:\n  read_file:\n    config:\n      directory: /Users/marcosalazar/code/dagster/js_modules/dagster-ui/packages/ui-core/src/ticks/tests\n      filename: DryRunRequestTable.test.tsx',
+          selector: {
+            jobName: 'saepe',
+            repositoryLocationName: 'testLocation',
+            repositoryName: 'testName',
+            assetSelection: [],
+            assetCheckSelection: [],
+            solidSelection: undefined,
+          },
+          mode: 'default',
+          executionMetadata: {
+            tags: [
+              {
```

Comment:
build

Reply:
could not get the build functions to work for request.variables part of MockedResponse, kept telling me the Expected variables didn't match the output no matter what I tried. I've looked at other <MockedResponse> examples in the codebase and none of them in test files have used the build functions in the request.variables section. Not sure the exact problem.